src/server.h

```diff
@@ -3010,6 +3010,7 @@ sds keyspaceEventsFlagsToString(int flags);
 #define DENY_LOADING_CONFIG (1ULL<<6) /* This config is forbidden during loading. */
 #define ALIAS_CONFIG (1ULL<<7) /* For configs with multiple names, this flag is set on the alias. */
 #define MODULE_CONFIG (1ULL<<8) /* This config is a module config */
+#define ALLOW_EMPTY_ARG_CONFIG (1ULL<<8) /* This config allow setting empty arg. */
```

Comment:
9?

Reply:
right.. thanks, I just saw that the test failed